main.go

```diff
@@ -26,6 +27,11 @@ func main() {
 	// }
 	// // debug
 	// fmt.Println(dataUnmarshalled)
+	// Quit command
+	var input string
+	if input == "quit" {
+		os.Exit(0)
+	}
```

Comment:
I would wait until #24  is done for this. So it actually can do something.

Otherwise that looks good.